Form-Controls/index.html

```diff
@@ -1,28 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Fashionable T.Shirt</h1>
+    </header>
+    <main>
+      <section class="container">
+        <div class="order-form">
+          <!-- Customer Name -->
+          <p>
+          <label for="FirstName">First Name*</label>
+          <input
+            type="text"
+            id="FirstName"
+            name="FirstName"
+            required
+            minlength="2"
+          />
+          </p>
+          <p>
+          <label for="LastName">Last Name*</label>
+          <input type="text" id="LastName" name="FirstName" required />
+          <br />
```

Comment:
I'm not a fan of using the br tag in this way. Spacing between elements should be handled by CSS, not HTML. Here's a massive article discussing when you should and shouldn't use br elements and what you can use instead. https://blog.hubspot.com/website/html-line-break